deploy/README.md

```diff
@@ -60,7 +60,9 @@ The following matrix displays the tested package versions for our Helm chart.
 
 Sumo Logic Helm Chart | Prometheus Operator | Fluent Bit | Falco  | Metrics Server
 |:-------- |:-------- |:-------- |:-------- |:--------
-0.14.0 - Latest | 8.2.0 | 2.8.1 | 1.1.1 | 2.7.0
+0.16.0 - Latest | 8.2.0 | 2.8.1 | 1.1.0 | 2.7.0
+0.15.0 - Latest | 8.2.0 | 2.8.1 | 1.1.0 | 2.7.0
+0.14.0 - Latest | 8.2.0 | 2.8.1 | 1.1.0 | 2.7.0
```

Comment:
@vsinghal13 @samjsong shouldn't we downgrade falco in the v0.14 and v0.15 and release them as patches as well?

Reply:
Also, we should change the support matrix to be just have `0.14.0 - Latest | 8.2.0 | 2.8.1 | 1.1.0 | 2.7.0` as the same applies to latest.

Reply:
Fixed.